### DISCUSSION TITLE
Rules to use the bootScore Logo

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#433] [Q&A]
I think I read somewhere that you put the bootScore logo under separate rules. That makes sense. Usually, brands are not covered by the licensing and licenses (here MIT). What are we allowed to do with the logos? Am I allowed to tease my articles/posts about bootScore with the bootScore logo? Do you have a general policy?

heartfully KR

**Comment by crftwrk** [comment]
No different rules, do-whatever-you-want with it.

**Reply by kreincke** [reply]
Thanks!
